docs/admin/install/installing-istio.md

```diff
@@ -2,6 +2,8 @@
 title: "Installing Istio for Knative"
 weight: 15
 type: "docs"
+aliases:
+  - /docs/install/installing-istio
```

Comment:
Not sure we want to do this now, but it might make sense to put this under `docs/admin/integrations/istio.md`, and (TODO, not in this PR) expand it to include how to use an existing Istio install with Knative.

Reply:
By TODO, I mean put the following text after the YAML front-matter:

```markdown
<!-- TODO: add documentation on how to reuse an existing Istio installation -->
```

Reply:
(See also #3524 )

Reply:
Yep, I still think there's too much stuff lumped into the install guide right now, but I didn't want to try to get too granular yet cos I'm going to iterate with more PRs once this is merged. I don't wanna make a PR impossible for folks to review by moving everything at once plus all the editing etc 😄

Reply:
Added comment, resolving for this PR